ui/src/components/NodePartitionTable.js

```diff
@@ -0,0 +1,243 @@
+//@flow
+import React, { useCallback } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { useTable } from 'react-table';
+import { useQuery } from 'react-query';
+import { ProgressBar, Loader } from '@scality/core-ui';
+import { fontSize, padding } from '@scality/core-ui/dist/style/theme';
+import {
+  queryNodeFSUsage,
+  queryNodeFSSize,
+} from '../services/prometheus/fetchMetrics';
+import CircleStatus from './CircleStatus';
+import { getAlerts } from '../services/alertmanager/api';
+import { getNodePartitionsTableData } from '../services/NodeVolumesUtils';
+import { intl } from '../translations/IntlGlobalProvider';
+
+const HeadRow = styled.tr`
+  width: 100%;
+  /* To display scroll bar on the table */
+  display: table;
+  table-layout: fixed;
+`;
+
+const Body = styled.tbody`
+  /* To display scroll bar on the table */
+  display: block;
+  overflow: auto;
+`;
+
+const SystemDeviceTableContainer = styled.div`
+  color: ${(props) => props.theme.brand.textPrimary};
+  padding: 1rem;
+  font-family: 'Lato';
+  font-size: ${fontSize.base};
+  border-color: ${(props) => props.theme.brand.borderLight};
+  table {
+    border-spacing: 0;
+
+    th {
+      font-weight: bold;
+      height: 56px;
+      border-bottom: 1px solid ${(props) => props.theme.brand.border};
+    }
+
+    td {
+      margin: 0;
+      padding: 0.5rem;
+      text-align: left;
+      padding: 5px;
+      height: 48px;
+      border-bottom: 1px solid ${(props) => props.theme.brand.border};
+      :last-child {
+        border-right: 0;
+      }
+    }
+    .sc-progressbarcontainer > div {
+      background-color: ${(props) => props.theme.brand.secondaryDark1};
+    }
+  }
+`;
+
+const Cell = styled.td`
+  overflow-wrap: break-word;
+`;
+
+const columns = [
+  {
+    Header: 'Health',
+    accessor: 'health',
+    cellStyle: { textAlign: 'center', width: '100px' },
+    Cell: (cellProps) => {
+      return (
+        <CircleStatus className="fas fa-circle" status={cellProps.value} />
+      );
+    },
+  },
+  {
+    Header: 'Partition path',
+    accessor: 'partitionPath',
+    cellStyle: { textAlign: 'left' },
+  },
+  {
+    Header: 'Usage',
+    accessor: 'usage',
+    cellStyle: { textAlign: 'center' },
+  },
+  {
+    Header: 'Size',
+    accessor: 'size',
+    cellStyle: { textAlign: 'right', width: '100px' },
+  },
+];
+
+const LoaderContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const NodePartitionTable = ({ instanceIP }: { instanceIP: string }) => {
```

Comment:
There are some tests missing for this component. IMO we should test success, loading and error variants of this component by mocking api calls.

Reply:
Will add the test for this component in the coming PR.